ckanext/ytp/comments/public/javascript/comments.js

```diff
@@ -1,3 +1,13 @@
 function ShowCommentForm(id){
     $("#" + id).removeClass('hidden');
 }
+
+jQuery(document).ready(function() {
```

Comment:
If dealing with jquery, wrap it like so to ensure $ is hooked to jQuery.
in google chrome dev console, use $x("") to test xpath 
```
(function ($) {
$(document).ready(function() {
$('.comment-container form .btn-cancel').on('click', function(e) {
        var form = $(this).closest('form');
        if (form) {
            form.addClass('hidden');
        }
    });
}(jQuery));
```

better yet if jquery is used throughout the js file, then wrapped the entire file with top and bottom line.